From bf91971e457588b90bcce8992617a93547a61a69 Mon Sep 17 00:00:00 2001
From: Tobias Schottdorf <tobias.schottdorf@gmail.com>
Date: Thu, 6 Oct 2016 03:47:33 -0400
Subject: [PATCH] sql: skip flaky TestReacquireLeaseOnRestart

This test changes the MaxOffset of a running node instead of starting one with
the desired clock offset. I suspect that that node is in fact pinging itself
and finds that it doesn't advertise the clock offset it has configured, but
I have not checked that.

Skipping this test since it's *extremely* flaky, but the fix should not be
too hard.
---
 sql/txn_restart_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sql/txn_restart_test.go b/sql/txn_restart_test.go
index d83fca3..efeba3a 100644
--- a/sql/txn_restart_test.go
+++ b/sql/txn_restart_test.go
@@ -1110,6 +1110,7 @@ func TestNonRetryableErrorOnCommit(t *testing.T) {
 // the clock, so that the transaction timestamp exceeds the deadline of the EndTransactionRequest.
 func TestReacquireLeaseOnRestart(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+	t.Skip("#9774")
 
 	var cmdFilters CommandFilters
 	cmdFilters.AppendFilter(checkEndTransactionTrigger, true)
